src/library_nodepath.js

```diff
@@ -17,18 +17,15 @@ addToLibrary({
     normalize: (path) => nodePath['normalize'](path),
     dirname: (path) => nodePath['dirname'](path),
     basename: (path) => nodePath['basename'](path),
-    join: function () {
-      return nodePath['join'].apply(null, arguments);
-    },
+    join: (...args) => nodePath['join'].apply(null, args),
```

Comment:
Since using this new syntax anyway, `nodePath['join'](...args)` might be better.

Reply:
Thanks!  Good idea.